src/main/java/mate/academy/spring/controller/CinemaHallController.java

```diff
@@ -0,0 +1,41 @@
+package mate.academy.spring.controller;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import mate.academy.spring.model.dto.CinemaHallRequestDto;
+import mate.academy.spring.model.dto.CinemaHallResponseDto;
+import mate.academy.spring.service.CinemaHallService;
+import mate.academy.spring.service.mapper.CinemaHallDtoMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/")
+public class CinemaHallController {
+    private final CinemaHallDtoMapper cinemaHallDtoMapper;
+    private final CinemaHallService cinemaHallService;
+
+    @Autowired
+    public CinemaHallController(CinemaHallDtoMapper cinemaHallDtoMapper,
+                                CinemaHallService cinemaHallService) {
+        this.cinemaHallDtoMapper = cinemaHallDtoMapper;
+        this.cinemaHallService = cinemaHallService;
+    }
+
+    @PostMapping("/cinema-halls")
```

Comment:
```suggestion
    @PostMapping
```
fix other controllers as well